internal/initialize/command.go

```diff
@@ -73,14 +67,44 @@ func NewCmdInit(t *terminal.Terminal) *cobra.Command {
 			return nil
 		},
 	}
-	cmd.Flags().StringVarP(&project, "project", "p", "", "Project Name")
-	cmd.RegisterFlagCompletionFunc("project", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	cmd.Flags().StringVarP(&name, "name", "p", "", "Project Name")
+	cmd.MarkFlagRequired("name")
+	cmd.RegisterFlagCompletionFunc("name", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		return getProjectNames(), cobra.ShellCompDirectiveNoSpace
 	})
 
 	return cmd
 }
 
+func NewCmdInit(t *terminal.Terminal) *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:   "init",
+		Short: "Initialize a Brev Project",
+		Long: `Use this to initialize a Brev project. Ex:
+		
+		// Init a new project in current directory
+		brev init
+	
+		`,
```

Comment:
Do you mind converting this to use the 'Example' built-in? I swept through in the other PR to make consistent use of the usage text
```suggestion
		Long: "Use this to initialize a Brev project."
		Example: `  // Init a new project in current directory
  brev init`,
```